docs/root/version_history/current.rst

```diff
@@ -38,6 +38,7 @@ Bug Fixes
 ---------
 *Changes expected to improve the state of the world and are unlikely to have negative effects*
 
+* aws_lambda: in JSON mode, the downstream response content-type header will now be set from the content-type entry in the JSON response's headers map, if present.
```

Comment:
nit: I don't think there's a "JSON mode" per se. It's either normal or passthrough.
So, this should probably read "unless passthrough is on, the downstream response..."